src/allmydata/test/test_storage.py

```diff
@@ -65,32 +75,32 @@ def make_lease(self):
         cancel_secret = os.urandom(32)
         expiration_time = time.time() + 5000
         return LeaseInfo(owner_num, renew_secret, cancel_secret,
-                         expiration_time, "\x00" * 20)
+                         expiration_time, b"\x00" * 20)
```

Comment:
Perhaps worth adding docs or a type check to `LeaseInfo` for this parameter (`nodeid`?) to make sure it is consistently bytes?

Assuming all callers are actually supplying bytes currently...

Reply:
Looks like it already has assert for that.